azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/DeploymentStatus.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.appservice;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.microsoft.azure.ProxyResource;
+import com.microsoft.rest.serializer.JsonFlatten;
+
+@JsonFlatten
+public class DeploymentStatus extends ProxyResource {
```

Comment:
Response class for `deploymentStatus`. Note this **will** get changed after swagger updated by service.